executor/executor.go

```diff
@@ -592,7 +623,8 @@ func (e *HashJoinExec) Fields() []*ast.ResultField {
 
 var batchSize = 128
 
-// Worker to get big table rows.
+// fetchBigExec fetches rows from big table in one background goroutine
+// and send the rows to multiple channels, multiple join workers will read those channels.
```

Comment:
-> and sends the rows to multiple channels which will be read by multiple join workers.